front/src/views/user/user-settings/user-settings.view.js

```diff
@@ -1,19 +1,40 @@
 import { MmtTextInput } from '@components';
 import { mapState } from 'vuex';
+import { VFileInput, VIcon } from 'vuetify/lib';
 const { emailRegex } = require('@/config/constants.json');
 
 export default {
   components: {
-    MmtTextInput
+    MmtTextInput,
+    VFileInput,
+    VIcon
   },
   data () {
     return {
       editedUserSettings: {},
       formValidity: true,
       emailIsAlreadyTaken: false,
+      fileUploadError: '',
       emailRules: [
         email => RegExp(emailRegex).test(email)
         || this.$t('userSettings.enterCorrectEmail')
+      ],
+      avatarRules: [
+        value => {
+          if (value && value.size > 1024 * 1024 * 2) {
+            return this.$t('userSettings.fileIsToBig');
+          }
+
+          if (value
+            && value.type !== 'image/png'
+            && value.type !== 'image/jpeg'
+            && value.type !== 'image/jpg'
+            && value.type !== 'image/bmp') {
+            return this.$t('userSettings.notAllowedContentType');
+          }
+
+          return true;
+        }
```

Comment:
you can use accept attribute accept https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept
see example on vuetify https://vuetifyjs.com/en/components/file-inputs/#accept-formats

Reply:
fixed